### PR TITLE
feat(cli): intro warning (31) support for row-level TTL

### PR DESCRIPTION
**Pull request by Druue**
closes #18733

pending latest engines in prisma/prisma

### CODE REVIEW

File: packages/migrate/src/__tests__/DbPull.test.ts [hunk 1]
```diff
@@ -2375,6 +2375,92 @@ describeIf(!process.env.TEST_SKIP_COCKROACHDB)('cockroachdb', () => {
...
+        generator client {
+          provider = "prisma-client-js"
+        }
+
+        datasource db {
+          provider = "cockroachdb"
+          url      = env("TEST_COCKROACH_URI_MIGRATE")
+        }
+
+        model ttl_test {
```
Comment (janpio) [inline packages/migrate/src/__tests__/DbPull.test.ts:2441]: No comment on the model?

**Review by janpio — APPROVED** [review]
Code looks good

File: packages/migrate/src/__tests__/DbPull.test.ts [hunk 2]
```diff
@@ -2550,6 +2550,92 @@ describeIf(!process.env.TEST_SKIP_COCKROACHDB)('cockroachdb', () => {
   })
 })
 
+describeIf(!process.env.TEST_SKIP_COCKROACHDB)('cockroachdb stopgaps', () => {
+  const connectionString = process.env.TEST_COCKROACH_URI_MIGRATE!.replace('tests-migrate', 'tests-migrate-db-pull')
```
Comment (janpio) [inline packages/migrate/src/__tests__/DbPull.test.ts:2554]: From CI:
```
FAIL src/__tests__/DbPull.test.ts
  ● Test suite failed to run

    TypeError: Cannot read property 'replace' of undefined

      2552 |
      2553 | describeIf(!process.env.TEST_SKIP_COCKROACHDB)('cockroachdb stopgaps', () => {
    > 2554 |   const connectionString = process.env.TEST_COCKROACH_URI_MIGRATE!.replace('tests-migrate', 'tests-migrate-db-pull')
           |                                                                    ^
      2555 |
      2556 |   const computeSetupParams = (warningCode: number, variant?: number): SetupParams => {
      2557 |     const setupParams: SetupParams = {

      at replace (src/__tests__/DbPull.test.ts:2554:68)
      at Object.<anonymous> (src/__tests__/DbPull.test.ts:2553:46)
```

Comment (Druue) [reply packages/migrate/src/__tests__/DbPull.test.ts]: Hopefully this fixes it :x

**Pull request merged**